package.json

```diff
@@ -85,18 +85,20 @@
   },
   "dependencies": {
     "@dr-kobros/react-webfont-loader": "^8.0.0",
+    "@testing-library/react": "^8.0.1",
     "@types/color": "^3.0.0",
+    "@types/react-syntax-highlighter": "^10.1.0",
+    "@types/react-window": "^1.7.0",
+    "@types/styled-components": "^4.1.12",
+    "@types/styled-system": "^4.0.0",
     "color": "^3.1.0",
+    "react-hooks-testing-library": "^0.5.1",
```

Comment:
Please put all dev dependencies (testing libraries, @types, etc), in, eh, devDependencies 😉 

Otherwise, all consumers of this package will download all of these packages in their node_modules, which they don't need.

Reply:
@Stefanwullems please move all dev dependencies

(someone knows whether GH supports adding a comment to multiple lines?)